Configuration/Extbase/Persistence/Classes.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+declare(strict_types=1);
+
+return [
+    \TTN\Tea\Domain\Model\Product\Tea::class => [
+        'properties' => [
+            'ownerUid' => ['fieldName' => 'owner'],
```

Comment:
Can you explain why you don't use the convention but use `ownerUid` and `owner`?

My thought was that the extension some day might switch to a full Owner Model and doesn't need to migrate the database column name in that case. And it is total fine within the database to use `owner` for both as it always will be the uid. But the PHP code should document its own type. Did you have other thoughts?

Reply:
By convention, we never have the `*_uid` suffix for associations in the DB column names. This is why I'm choosing `owner` instead of `owner_uid`.

On the PHP side, things are different. An `owner` implicates that this is an object (in this case, a FE user model), not an int (a UID). To make it clear that we're dealing with the UID and not the model itself (and hence reduce the amount of surprises in the code), I've chosen the name `ownerUid` instead of `owner`. (Also, this indeed leaves the way open for us to switch to an association to a model using the association name `owner` later if the need arises.)

Reply:
Thanks for clarification, that resolves the conversation for me.